ODLA/include/ODLA/odla_common.h

```diff
@@ -134,6 +134,14 @@ typedef enum {
   ODLA_FAILURE,
 } odla_status;
 
+//! \brief BF16 mode
+typedef enum {
+  BF16_DISABLE,
+  BF16_ACCURACY_MODE,
+  BF16_PERFORMACE_MODE,
+  BF16_AUTO_MODE,
+} odla_bf16_mode;
+
```

Comment:
This is execution related and is used together with "ODLA_BF16_MODE"
 how about define in "odla_compute.h" ?

Reply:
ok,  better to put in compute.h